services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -307,225 +394,212 @@ public Object answer() throws Throwable {
      */
     @Test
     public void processRequestToSubmitTimeOut() {
-        LOGGER.debug("Timeout scenario");
+        Logger logger = (Logger) LoggerFactory.getLogger(TargetApplicationSubmissionService.class);
+        logger.setLevel(DEBUG);
 
-        final String sdtRequestRef = "TEST_1";
+        final String sdtRequestRef = TEST_1;
```

Comment:
The sdtRequestRef variable is redundant and can be removed.  The TEST_1 constant can be used in place of it.

Reply:
done